### PR TITLE
Added contact form

### PR DESCRIPTION
**Pull request by adeola003**
To achieve the expected result, I have been through the following steps:
1. I opened the previous project;
2. I created a new branch named "add_contact_form";
3. I created a new section for the form in my HTML file;
4. I added the form and then styled it in CSS trying to match as much as possible with the template.
5. I added @Atifzada as a collaborator;
6. He added Action and method in the form;
7. He added validations as per the requirements;
8. He added placeholders;
9. Then validations were checked.

### CODE REVIEW

**Review by osamaashraf6 — COMMENTED** [review]
Hi @adeola003 😊, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🆘💀

- You have submitted a project, that no partners contributed at the project, so everyone has contributed to the project to make the pull request valid, so kindly try to make your partner a collaborator and let him/her write a commit with you at the pull request and with that the pull request will be valid 😃

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -289,6 +289,36 @@ <h2 class="header-text">About <br>Myself</h2>
       </div>
     </div>
   </section>
+  <section id="Contact" class="contact-me">
+    <h2 class="intro">Contact me</h2>
+    <p class="intro-contact">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it. </p>
+    <form action="https://formspree.io/f/mlekbpkb" method="POST">
+      <ul>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] Nice work you did so far 👏 . However, Kindly use flexbox to layout elements of the form filed. This give you extra flexibility 👍 

## Instructions :
image

File: index.html [hunk 2]
```diff
@@ -289,6 +289,36 @@ <h2 class="header-text">About <br>Myself</h2>
...
+        <li>
+          <label for="msg">
+            <textarea id="msg" name="user_message" maxlength="500" required placeholder="Write your message"></textarea>
+          </label>
+        </li>
+        <li>
+          <label for="button" class="getInTouch">
+            <input type="submit" id="button" name="submit" value="Get in touch" />
+          </label>
+        </li>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] With refrence to the figma design, kindly, adjust the styling of your button, to look same as that given below ;:
## Your design:
image

## Figma:
image

File: style.css [hunk 3]
```diff
@@ -394,33 +368,76 @@ span {
...
-  margin-bottom: 30px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  transition-duration: 0.4s;
```
Comment (Shedrack-Sunday) [inline style.css]: - [ ] Kindly adhere to the hover color pattern given , as your current reactions does not tally with the template olors below :

## The hover colors:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @adeola003 @Atifzada    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅All linters are checked
✅ Descriptive commit messages 
  
Required Changes ♻️

- [x] The slight change needed has to do with a bit of styling of the buttons.

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by roshan-bajgain — APPROVED** [review]
Hi @adeola003, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**